Default.sublime-keymap

```diff
@@ -0,0 +1,3 @@
+[
+    { "keys": ["super+shift+o"], "command": "side_bar_folders_quick_switch" }
```

Comment:
> Are you suggesting that I should remove the keymap file all together and leave it up to the user to configure the shortcut after installing the plugin?

Basically, yeah.  Just document an example.

For instance, I have a personal shortcut already configured for <kbd>super</kbd> + <kbd>shift</kbd> + <kbd>o</kbd>.  There are frankly too many plugins that assume everyone wants their shortcuts to be defined by the plugins.  There is no way for a plugin author to be aware of all the plugins trying to use the same shortcut.  It causes confusion.  It is much better just to provide an example keymap and let the user do it.

If this got pulled as it is now, it is guaranteed to break me and probably a number of others.  I think it is much better to break no one, and the people who want a shortcut can simply copy and paste (and possibly tweak) an example if desired.

The only exception for including keymaps is if they have a context assigned to them that limits them to a very particular setting.  For instance, I have plugins that I have written that enable keymaps only within a special view that is plugin specific, or a keymap that is only enabled in the quick panel for a specific command.  By narrowing the scope, I conflict with nothing, but global keymaps will certainly conflict with a number of peoples setups.